ggu/main.c

```diff
@@ -405,10 +402,12 @@ ggu_proc(void *arg)
 		}
 	} else {
 		while (likely(!exiting)) {
-			struct ggu_request *reqs[GGU_REQ_BURST_SIZE];
+			unsigned int mailbox_burst_size =
+				ggu_conf->mailbox_burst_size;
```

Comment:
Move variable `mailbox_burst_size` declaration and initialization just before the `while` loop to avoid recomputing it every iteration.